typescript/sdk/src/consts/chainMetadata.ts

```diff
@@ -130,20 +165,78 @@ export const ChainMetadataSchema = z.object({
         family: z.nativeEnum(ExplorerFamily).optional(),
       }),
     )
-    .optional(),
+    .optional()
+    .describe(`Block explorers for the chain.`),
   blocks: z
     .object({
-      confirmations: z.number(),
-      reorgPeriod: z.number().optional(),
-      estimateBlockTime: z.number().positive().optional(),
+      confirmations: z
+        .number()
+        .describe(
+          `Number of blocks to wait before considering a transaction confirmed.`,
+        ),
+      reorgPeriod: z
+        .number()
+        .optional()
+        .describe(
+          'Number of blocks before a transaction has a near-zero chance of reverting.',
+        ),
+      estimateBlockTime: z
+        .number()
+        .positive()
+        .optional()
+        .describe('Rough estimate of time per block in seconds.'),
     })
-    .optional(),
+    .optional()
+    .describe(`Block settings for the chain/deployment.`),
   transactionOverrides: z.object({}).optional(),
   gasCurrencyCoinGeckoId: z.string().optional(),
   gnosisSafeTransactionServiceUrl: z.string().optional(),
   isTestnet: z.boolean().optional(),
 });
 
+export const HyperlaneDeploymentArtifacts = z.object({
```

Comment:
Not sure if these deployment artifacts are getting merged into the prior schema but I know we talked about having them flattened into the chain object.

Reply:
On line 219 it gets merged into `AgentChainMetadataSchema`